maps_dashboards/public/components/maps_list/maps_list.tsx

```diff
@@ -9,42 +9,49 @@ import { I18nProvider } from '@osd/i18n/react';
 import { EuiPage, EuiPageBody, EuiPageContentBody } from '@elastic/eui';
 import { CoreStart } from 'opensearch-dashboards/public';
 import { TableListView } from '../../../../../src/plugins/opensearch_dashboards_react/public';
+import { MapSavedObjectAttributes } from '../../../common';
 
 export const MapsList = (props: {
   notifications: CoreStart['notifications'];
   http: CoreStart['http'];
+  savedObjectsClient: CoreStart['savedObjects']['client'];
 }) => {
-  const { http } = props;
-  const find = async (num: number) => {
-    const res = await http.post('/api/maps-dashboards/example');
-    return {
-      total: num,
-      hits: res.hits,
-    };
-  };
+  const { http, savedObjectsClient } = props;
 
   const tableColumns = [
     {
-      field: 'title',
+      field: 'attributes.title',
       name: i18n.translate('maps.listing.table.titleColumnName', {
         defaultMessage: 'Title',
       }),
       sortable: true,
     },
     {
-      field: 'description',
+      field: 'attributes.description',
       name: i18n.translate('maps.listing.table.descriptionColumnName', {
         defaultMessage: 'Description',
       }),
       sortable: true,
     },
   ];
 
-  const createItem = useCallback(() => {
+  const createMap = useCallback(() => {
     window.location.href = http.basePath.prepend('/app/maps-dashboards/#/create-map');
```

Comment:
Ideally this should just be a link and not a callback function. Is there a reason for that?

If it can be a link, use `<EUIButton href="yourpath" >Create Map</EUIButton>` should render a link as a button

if it cant be a link, use `application.navigateToUrl`

Reply:
Updated to use `application.navigateToUrl`.